packages/manager/modules/at-internet-configuration/src/index.js

```diff
@@ -28,6 +28,7 @@ angular
     ) => {
       atInternetProvider.setEnabled(trackingEnabled);
       atInternetProvider.setDebug(!trackingEnabled);
+      atInternetProvider.setRegion(Environment.getRegion());
```

Comment:
This package has no dependency on `@ovh-ux/manager-config`.
And we should avoid to add a dependency on `Environment`, as https://github.com/ovh/manager/pull/4505 will remove it as a singleton.

I think if we really need the region we should have a provider to configure it.

Reply:
> This package has no dependency on @ovh-ux/manager-config.

So we are lucky :-) as it's also used at: https://github.com/ovh/manager/blob/9bba0f475e2b99ac7d9746a3e0981de95e4f8971/packages/manager/modules/at-internet-configuration/src/index.js#L78

> I think if we really need the region we should have a provider to configure it.

Ok, how to get the correct region? All apps will require to configure the `at-internet-configuration` module, I'm right?

Reply:
Ok I updated it to use coreConfig instead from `manager-core` (https://github.com/ovh/manager/blob/feat%2Fufrontend-environment/packages/manager/modules/at-internet-configuration/src/index.js#L81).

`coreConfig` is already in the codebase, so it's already ok to use `coreConfig.getRegion()` instead of `Environment.getRegion()`

Reply:
Done. `manager-config` has been replaced by coreConfig